VimeoUpload.xcworkspace/contents.xcworkspacedata

```diff
@@ -2,15 +2,15 @@
 <Workspace
```

Comment:
Eh some Xcode metadata. I think they sometimes get changed when you modify the project structure.